### PR TITLE
Replace hardcoded NSG suffixes with constants

### PR DESCRIPTION
**Pull request by sjkingo**
Implements #72

### CODE REVIEW

File: pkg/deploy/generator/const.go [hunk 1]
```diff
@@ -28,3 +28,9 @@ const (
 	fileRPDevelopmentPredeploy = "rp-development-predeploy.json"
 	fileRPDevelopment          = "rp-development.json"
 )
+
+// Network security group constants
+const (
+	NSGControlPlaneSuffix = "-controlplane-nsg"
+	NSGNodeSuffix         = "-node-nsg"
+)
```
Comment (mjudeikis) [inline pkg/deploy/generator/const.go:36]: I think it worth merging them with "Deployment const". Name is misleading "Deployments" as all of them are specific resource constants used in Deployments. some of those values are exported to use externally, some no.

Comment (jim-minter) [reply pkg/deploy/generator/const.go:36]: Please put these in pkg/util/subnet.

**Comment by sjkingo** [comment]
Have moved constants and rebased

**Comment by jim-minter** [comment]
rebased

**Pull request merged**